system/lib/compiler-rt/lib/lsan/lsan_common_emscripten.cpp

```diff
@@ -142,7 +140,7 @@ static void ProcessThreadsCallback(ThreadContextBase *tctx, void *arg) {
 
     // We can't get the SP for other threads to narrow down the range, but we
     // we can for the current thread.
-    if (tctx->tid == GetCurrentThreadId()) {
```

Comment:
I noticed now that this removed a reference to `GetCurrentThreadId` and the first part of this diff removes a `GetCurrentThread` impl (without `Id` as a suffix). Is that right?

Reply:
Yes.  `GetCurrentThreadId` is an inline function implemented in terms of `GetCurrentThread`:

So the usage of `GetCurrentThreadId` was what triggered the need to the `lsan::GetCurrentThread` hack which was wong in the pure lsan build.

Reply:
https://github.com/emscripten-core/emscripten/blob/4eb604ec7415bc227891090fe91acd6a8e60b744/system/lib/compiler-rt/lib/lsan/lsan_thread.h#L57-L60